CMakeLists.txt

```diff
@@ -77,18 +80,7 @@ set(EP_BASE ${CMAKE_BINARY_DIR}/ThirdParty)
 # Determine common toolchain settings, compiler and link flags
 include(CommonCompilerConfig)
 
-# Set up external dependencies.
-#
-# This will try find_library and find_program and if that fails use
-# ExternalProject to fetch and build the libraries.
-#
-# Sets the following variables:
-#   CAPNP_LINK_LIBRARIES
```

Comment:
The problem is that things defined in one subdirectory (external) need to be used in another (src). That is a difficult relationship to figure out and it is enable in this file so I thought it would be nice to explicitly call out the items defined in external that we are intentionally exposed to src. And we don't have to include everything, just the things that we want src to know about. Perhaps this explanation could be converted into a better comment?